src/Symfony/Component/HttpFoundation/Request.php

```diff
@@ -280,7 +280,7 @@ public static function createFromGlobals()
 
         $request = self::createRequestFromFactory($_GET, $_POST, array(), $_COOKIE, $_FILES, $server);
 
-        if (0 === strpos($request->headers->get('CONTENT_TYPE'), 'application/x-www-form-urlencoded')
+        if (in_array($request->headers->get('CONTENT_TYPE'), array('application/x-www-form-urlencoded', 'multipart/form-data'))
```

Comment:
Missed that point sorry ! I need to manualy  parse request content. I will try with an event listener instead, to use property accessor.